dts/msm8916/msm8916-samsung-r00.dts

```diff
@@ -15,7 +15,16 @@
 		compatible = "samsung,gt58lte-aus", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-bootloader = "T355Y*";
 	};
+        on7lte-kor {
+		model = "Samsung Galaxy On7 2015 (SM-G600S)";
+		compatible = "samsung,on7lte-kor", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "G600S*";
 
+		samsung,muic-reset {
+			i2c-gpio-pins = <105 106>;
+			i2c-address = <0x25>;
+		};
+	};
 	coreprimelte {
```

Comment:
Let's keep an empty line between nodes.

```suggestion
	};

	coreprimelte {
```

Reply:
i dont know how to use github(im newbie) how can i update readme

Reply:
Just update the readme file. You can look at other commits that add new devices in the commit history :)

Reply:
sorry
i dont know how to add commit on this pull request

Reply:
I suppose you have already made this change locally and tested that the lk2nd with your changes works on your device. Please make sure these changes look correct (i.e. tabs are used for indentation); make the readme change from here. Create a new branch with the same name as the one used in #260 Then make a new commit (`git add .` then `git commit`, see how other commits write the subject and do it similarly). After that you would need to add your fork as a remote (`git remote add <some name> <your url>`) and update the branch, overwriting it and discarding remote changes (`git push --force`)

Reply:
thanks ill try it

Reply:
pls merge, i edit readme in patch-1

Reply:
Please note I've asked you to make some changes. I'd also appreciate if you can squash all commits into one and give it a fitting subject. (See other commits in the history, you can use `git rebase --interactive` to squash the changes)

Reply:
Hello? Pls merge

Reply:
You need to solve all threads before merging.
Please join [#msm8916:postmarketos.org](https://app.element.io/#/room/#msm8916:postmarketos.org) if you have questions.

Reply:
> You need to solve all threads before merging.

Or @TravMurav or other maintainer could just amend the change and merge on behalf of the author to speedup.